### PR TITLE
Adding get function for MOI.ConstraintSet for NonposToNonnegBridge

### PR DESCRIPTION
**Pull request by yashcodes**
Adding a get function to for `MOI.ConstraintSet` attribute for `FlipSignBridge` . Test is also added.

### CODE REVIEW

File: src/Bridges/Variable/flip_sign.jl [hunk 1]
```diff
@@ -48,6 +48,9 @@ function MOI.delete(model::MOI.ModelLike, bridge::FlipSignBridge, i::IndexInVect
 end
 
 # Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintSet, bridge::FlipSignBridge)
+    return MOI.Nonpositives(length(bridge.flipped_variables))
```
Comment (blegat) [inline src/Bridges/Variable/flip_sign.jl:52]: This is only valid for `NonposToNonnegBridge`, you can do
```julia
..., bridge::FlipSignBridge{T, S1}) where {T, S1}
return S1(...)
```

File: test/Bridges/Variable/flip_sign.jl [hunk 2]
```diff
@@ -24,6 +24,9 @@ config = MOIT.TestConfig()
         vis = MOI.get(bridged_mock, MOI.ListOfVariableIndices())
         y = vis[4]
         @test y.value == -1
+        
+        con = MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorOfVariables, MathOptInterface.Nonpositives}())[1]
```
Comment (blegat) [inline test/Bridges/Variable/flip_sign.jl:28]: Use `test_models_equal` to test the getter of `ConstraintSet`. See
https://github.com/JuliaOpt/MathOptInterface.jl/blob/5b25a050d99efb1a57a796d4ca2ffc5cdcf44e17/test/Bridges/Constraint/geomean.jl#L29-L72

File: src/Bridges/Variable/flip_sign.jl (hunk 1 above)
Comment (yashcodes) [reply src/Bridges/Variable/flip_sign.jl]: Yeah cool, that would much nicer.

Comment (yashcodes) [reply src/Bridges/Variable/flip_sign.jl]: But I think abstract type `FlipSignBridge{T, S1}` has only `NonposToNonnegBridge` as a subtype, there is no `NonnegToNonposBridge`.So, I think using `MOI.Nonpositives(length(bridge.flipped_variables))` would also be ok.

File: test/Bridges/Variable/flip_sign.jl [hunk 3]
```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
         @test MOI.get(bridged_mock, MOI.VariablePrimalStart(), y) == 1
     end
 
+    @testset "Test Mock model" begin
```
Comment (blegat) [inline test/Bridges/Variable/flip_sign.jl:36]: Put this testset inside the `lin2v` testset.

File: test/Bridges/Variable/flip_sign.jl [hunk 4]
```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
...
+        minobjective: 3*a + -2*b + -4*c
+        """
+        model = MOIU.Model{Float64}()
+        MOIU.loadfromstring!(model, s)
+        MOIU.test_models_equal(mock, model, var_names, ["cd", "cb", "cc", "cex"])
+    end
+
+    @testset "Test Bridged model" begin
+        var_names = ["x", "y", "z", "w"]
+        MOI.set(bridged_mock, MOI.VariableName(), MOI.get(bridged_mock, MOI.ListOfVariableIndices()), var_names)
```
Comment (blegat) [inline test/Bridges/Variable/flip_sign.jl:64]: Move this outside and before both testsets

File: test/Bridges/Variable/flip_sign.jl [hunk 5]
```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
         @test MOI.get(bridged_mock, MOI.VariablePrimalStart(), y) == 1
     end
 
+    @testset "Test Mock model" begin
+        var_names = ["a", "b", "c", "d"]
+        MOI.set(mock, MOI.VariableName(), MOI.get(mock, MOI.ListOfVariableIndices()), var_names)
```
Comment (blegat) [inline test/Bridges/Variable/flip_sign.jl:38]: Only set the name of the bridged variables, the other variables should have the names `x`, `z` and `w` already set.

File: test/Bridges/Variable/flip_sign.jl [hunk 6]
```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
...
+
+        con_d = MOI.get(mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorOfVariables, MathOptInterface.Zeros}())[1]
+        con_bc = MOI.get(mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorOfVariables, MathOptInterface.Nonnegatives}())
+        con_ex = MOI.get(mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorAffineFunction{Float64}, MathOptInterface.Zeros}())[1]
```
Comment (blegat) [inline test/Bridges/Variable/flip_sign.jl:42]: Same than for variable names, you should not set the name here of the constraint that are not bridged. It should be set only once for both `mock` and `bridged_mock`.

File: test/Bridges/Variable/flip_sign.jl (hunk 3 above)
Comment (yashcodes) [reply test/Bridges/Variable/flip_sign.jl:36]: Only this or both testsets inside in line2v?

**Pull request merged**